tests/rptest/tests/data_transforms_test.py

```diff
@@ -672,4 +684,214 @@ def test_tunable_configs(self):
+                        endpoint=MetricsEndpoint.METRICS))
+                for m in samples['events_total']:
+                    xfm_name = m['labels']['transform_name']
+                    totals[xfm_name] = totals.get(xfm_name, 0) + m['value']
+                dropped += sum(
+                    [m['value'] for m in samples["events_dropped_total"]])
+            self.logger.debug(json.dumps(totals, indent=1))
+            return (totals, dropped)
+
+        def get_total_events() -> tuple[int, int]:
+            tot = 0
+            totals, dropped = get_total_events_per_xform()
+            for k in totals:
+                tot += totals[k]
+            self.logger.debug(f"Cluster total log events: {tot}")
+            return (tot, dropped)
+
+        n_events_expected = N_PER_TP * len(its)
+        wait_until(lambda: get_total_events()[0] == n_events_expected,
+                   timeout_sec=30,
+                   backoff_sec=5,
+                   err_msg=f"never got all the events")
+
+        totals, dropped = get_total_events_per_xform()
+        assert len(totals) == len(its)
+        for name in totals:
+            assert totals[
+                name] == N_PER_TP, f"Expected {N_PER_TP} log events from transform {name}, got{totals[name]}"
+        assert dropped == 0, f"Expected no dropped events, got {dropped}"
+
+    @cluster(num_nodes=3)
+    def test_manager_metrics_present(self):
+        it, _ = self.setup_identity_xform(self.topics[0], self.topics[1])
+
+        def has_manager_metrics(node, endpoint):
+            metrics_samples = self.get_metrics_from_node(node,
+                                                         self.MANAGER_METRICS,
+                                                         endpoint=endpoint)
+            assert metrics_samples is not None
+            return sorted(metrics_samples.keys()) == sorted(
+                self.MANAGER_METRICS)
+
+        assert all(
+            [
+                has_manager_metrics(n, MetricsEndpoint.METRICS)
+                for n in self.redpanda.nodes
+            ]
+        ), "Expected all nodes to export manager metrics on the internal endpoint"
+
+        assert not any(
+            [
+                has_manager_metrics(n, MetricsEndpoint.PUBLIC_METRICS)
+                for n in self.redpanda.nodes
+            ]
+        ), "Expected no node to export manager metrics on the internal endpoint"
+
+    @cluster(num_nodes=3)
+    def test_manager_metrics_values(self):
+        self.logger.debug("Set flush interval to 1h so buffers fill up")
+        self.redpanda.set_cluster_config(
+            {'data_transforms_logging_flush_interval_ms': 60 * 60 * 1000})
+
+        it, ot = self.setup_identity_xform(
+            self.topics[0],
+            self.topics[1],
+            name=f"logger-xform",
+        )
+
+        self.logger.debug(
+            "Produce enough data to make a noticeable dent but not so much as to trip the buffer LWM"
```

Comment:
We can still get LWM with retries in transforms right? Let's be extra careful.

Reply:
Yeah, I hadn't really taken that into account. Will do a bit of rework. Incidentally, I've run these on the order of 100s of times w/o issue. I wonder whether there's a way to encourage or trigger retries in the test?

Reply:
run the stress utility while ducktape is running? CI is not as stable as an environment as locally, but yeah I have had the same concerns. I think the perf team was looking into what CPUs our tests run on to try and track this.

Reply:
Is this comment/log still valid?

Reply:
Sort of. We still perform the test, but the log is not very informative.